.github/workflows/publish.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       uses: actions/setup-node@main
       with:
         node-version: ${{ matrix.node-version }}
-    - run: yarn install --frozen-lockfile
```

Comment:
I think these should stay in. We want a predetermined dependency versions when we are publishing.